features/step_definitions/bin.js

```diff
@@ -319,6 +319,11 @@ module.exports = function() {
     if (this.isDryRun()) { return callback(); }
 
     var world = this;
+    var actualJson = world[stream];
+
+    if (expectedJson.length === 0 && actualJson.length === 0) {

```

Comment:
@peter-reperger I guess we wouldn't need this if the output from parallel-cucumber-js when no scenarios were executed was always an empty array rather than an emptry string?